ports/raspberrypi/gen_stage2.py

```diff
@@ -0,0 +1,82 @@
+import sys
+import cascadetoml
+import pathlib
+import typer
+from jinja2 import Template
+
+
+def main(input_template: pathlib.Path, output_path: pathlib.Path, skus: str = typer.Argument("")):
+    if ", " in skus:
+        skus = skus.split(", ")
```

Comment:
There are one or two board files (not in `raspberrypi`) that don't use `, `, but just `,`. How about testing for `,` and then split and trim, or use `re.split()`.